comps/cores/mega/gateway.py

```diff
@@ -156,9 +156,12 @@ def __init__(self, megaservice, host="0.0.0.0", port=8888):
 
     async def handle_request(self, request: Request):
         data = await request.json()
+        print("data in handle request", data)
         stream_opt = data.get("stream", True)
         chat_request = ChatCompletionRequest.parse_obj(data)
+        print("chat request in handle request", chat_request)
         prompt = self._handle_message(chat_request.messages)
+        print("prompt in gateway", prompt)
```

Comment:
Same here

Reply:
I just added GraphRagGateway to the end of the file and this belongs to chatqna gateway. maybe they kept those for a reason, I wont touch.